AppInspector.CLI/Writers/AnalyzeHtmlWriter.cs

```diff
@@ -56,23 +55,23 @@ private void WriteHtmlResult()
             RenderResultsSafeforHTML();
 
             //Grab any local css and js files that are needed i.e. don't have hosted URL's or are proprietary
-            string allCSS = "<style>\n" + MergeResourceFiles(Path.Combine(Utils.GetPath(Utils.AppPath.basePath), "html","resources","css"));
-            string allJS = "<script type=\"text/javascript\">\n" + MergeResourceFiles(Path.Combine(Utils.GetPath(Utils.AppPath.basePath),"html","resources","js"));
+            string allCSS = "<style>\n" + MergeResourceFiles(Path.Combine(Utils.GetPath(Utils.AppPath.basePath), "html", "resources", "css"));
+            string allJS = "<script type=\"text/javascript\">\n" + MergeResourceFiles(Path.Combine(Utils.GetPath(Utils.AppPath.basePath), "html", "resources", "js"));
```

Comment:
nit; but "include everything" is a good target for supply chain attacks.

Reply:
Agreed but I think I'll put off any modifications to the html output til later.